### ISSUE TITLE
FAILURE: Build failed with an exception, could not find flipper 0.158.0 for Android

### ISSUE THREAD

**Issue by shensven** [facebook/flipper#3990]
## 🐛 Bug Report

``` sh
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:testDebugUnitTest'.
> Could not resolve all task dependencies for configuration ':app:debugUnitTestRuntimeClasspath'.
   > Could not find com.facebook.flipper:flipper:0.158.0.
     Searched in the following locations:
       - file:/Users/shensven/Documents/GitHub/CrazyThursday/node_modules/react-native/android/com/facebook/flipper/flipper/0.158.0/flipper-0.158.0.pom
       - file:/Users/shensven/Documents/GitHub/CrazyThursday/node_modules/jsc-android/dist/com/facebook/flipper/flipper/0.158.0/flipper-0.158.0.pom
       - https://repo.maven.apache.org/maven2/com/facebook/flipper/flipper/0.158.0/flipper-0.158.0.pom
       - https://dl.google.com/dl/android/maven2/com/facebook/flipper/flipper/0.158.0/flipper-0.158.0.pom
       - https://www.jitpack.io/com/facebook/flipper/flipper/0.158.0/flipper-0.158.0.pom
     Required by:
         project :app
         project :app > project :react-native-flipper
   > Could not find com.facebook.flipper:flipper-network-plugin:0.158.0.
     Searched in the following locations:
       - file:/Users/shensven/Documents/GitHub/CrazyThursday/node_modules/react-native/android/com/facebook/flipper/flipper-network-plugin/0.158.0/flipper-network-plugin-0.158.0.pom
       - file:/Users/shensven/Documents/GitHub/CrazyThursday/node_modules/jsc-android/dist/com/facebook/flipper/flipper-network-plugin/0.158.0/flipper-network-plugin-0.158.0.pom
       - https://repo.maven.apache.org/maven2/com/facebook/flipper/flipper-network-plugin/0.158.0/flipper-network-plugin-0.158.0.pom
       - https://dl.google.com/dl/android/maven2/com/facebook/flipper/flipper-network-plugin/0.158.0/flipper-network-plugin-0.158.0.pom
       - https://www.jitpack.io/com/facebook/flipper/flipper-network-plugin/0.158.0/flipper-network-plugin-0.158.0.pom
     Required by:
         project :app
   > Could not find com.facebook.flipper:flipper-fresco-plugin:0.158.0.
     Searched in the following locations:
       - file:/Users/shensven/Documents/GitHub/CrazyThursday/node_modules/react-native/android/com/facebook/flipper/flipper-fresco-plugin/0.158.0/flipper-fresco-plugin-0.158.0.pom
       - file:/Users/shensven/Documents/GitHub/CrazyThursday/node_modules/jsc-android/dist/com/facebook/flipper/flipper-fresco-plugin/0.158.0/flipper-fresco-plugin-0.158.0.pom
       - https://repo.maven.apache.org/maven2/com/facebook/flipper/flipper-fresco-plugin/0.158.0/flipper-fresco-plugin-0.158.0.pom
       - https://dl.google.com/dl/android/maven2/com/facebook/flipper/flipper-fresco-plugin/0.158.0/flipper-fresco-plugin-0.158.0.pom
       - https://www.jitpack.io/com/facebook/flipper/flipper-fresco-plugin/0.158.0/flipper-fresco-plugin-0.158.0.pom
     Required by:
         project :app

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.3.3/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 20s
6 actionable tasks: 6 up-to-date
```

## To Reproduce

Update `FLIPPER_VERSION` from `0.157.0` to `0.158.0`, then run `./gradlew test`
```properties
# android/gradle.properties
# Project-wide Gradle settings.
#
# ...
#
# Version of flipper SDK to use with React Native
FLIPPER_VERSION=0.158.0
``` 
Also see this error reporting in GitHub Actions
https://github.com/shensven/Crazy-Thursday/runs/7780660656?check_suite_focus=true

**Comment by BLOCKMATERIAL** [comment]
Same issue

**Comment by thiagosan252** [comment]
I also had the same problem

React: 18.0.0
React-native: 0.69.3
Flipper: 0.158.0

The problem is due to the Maven repository not containing the file in that version, only up to version 0.157.0. In my case it solved by installing the lower version (0.157.0).
Unable to download non-existent version.

<img width="657" alt="Captura de Tela 2022-08-15 às 09 57 11" src="https://user-images.githubusercontent.com/63353000/184639162-16a3a10f-7689-4dfd-80bb-27f8bea8b826.png">

**Comment by NightWarrior** [comment]
Looking through maven repo and finding the latest version worked for me, here is the link if anyone wants to use this method and not have to type the link by himself: https://repo.maven.apache.org/maven2/com/facebook/flipper/flipper/

> I also had the same problem
> 
> React: 18.0.0 React-native: 0.69.3 Flipper: 0.158.0
> 
> The problem is due to the Maven repository not containing the file in that version, only up to version 0.157.0. In my case it solved by installing the lower version (0.157.0). Unable to download non-existent version.
> 
> <img alt="Captura de Tela 2022-08-15 às 09 57 11" width="657" src="https://user-images.githubusercontent.com/63353000/184639162-16a3a10f-7689-4dfd-80bb-27f8bea8b826.png">
